Sources/Bow/Typeclasses/Applicative.swift

```diff
@@ -1,77 +1,199 @@
 import Foundation
 
+/// An Applicative Functor is a `Functor` that also provides functionality to embed pure expressions, and sequence computations and combine their results.
```

Comment:
I'd change this to the following, for consistency with the docs of `pure`:

```suggestion
/// An Applicative Functor is a `Functor` that also provides functionality to lift pure expressions, and sequence computations and combine their results.
```